addons/transifex/models/models.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models
+
+class BaseModel(models.AbstractModel):
+    _inherit = 'base'
+
+    def get_field_translations(self, field_name, langs=None):
+        """ get model/model_term translations for records with transifex url
+        :param str field_name: field name
+        :param list langs: languages
+
+        :return: (translations, context) where
+            translations: list of dicts like [{"lang": lang, "source": source_term, "value": value_term,
+                    "module": module, "transifexURL": transifex_url}]
+            context: {"translation_type": "text"/"char", "translation_show_source": True/False}
+        """
+        translations, context = super().get_field_translations(field_name, langs=langs)
+        self._update_transifex_url(translations)
+        return translations, context
+
+    def _update_transifex_url(self, translations):
```

Comment:
Having the same method name for two different things, and having the base method invoking an override of itself is confusing.  You could simply inline the code of this one in `get_field_translations()` above.